app/views/choice_page.scala.html

```diff
@@ -20,7 +20,7 @@
 @import forms.Choice._
 @import views.components.BackButton
 @import views.Title
-@import components.gds.{errorSummary, gds_main_template, pageTitle}
+@import components.gds.{childrenUcrs, errorSummary, gds_main_template, pageTitle}
```

Comment:
Why does the choice page need this import?

Reply:
I checked before how the table looks like, this page is removed for now, I forgot to clean the import. Good spot!